src/main/java/mate/academy/springboot/datajpa/model/Product.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.springboot.datajpa.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString

```

Comment:
Using this annotation you still may receive LazyInitializationException. This is great that you are using lazy fetching for a category, but please try to disable the open session in view, and then call `product.toString()` in some place of your code. As a result, you should receive LazyInitializationException